panelapp/models.py

```diff
@@ -0,0 +1,53 @@
+from django.db import models
+from django.db.models import JSONField
+
+from seqr.models import LocusList as SeqrLocusList, LocusListGene as SeqrLocusListGene
+
+
+class PaLocusList(models.Model):
+    """PanelApp extension of seqr.models.LocusList."""
+
+    seqr_locus_list = models.OneToOneField(
+        SeqrLocusList,
+        on_delete=models.CASCADE,
+        primary_key=True,
+    )
+
+    panel_app_id = models.IntegerField(null=False, blank=False, unique=True)
+    disease_group = models.TextField(null=True, blank=True)
+    disease_sub_group = models.TextField(null=True, blank=True)
+    status = models.TextField(null=True, blank=True)
+    version = models.TextField(null=True, blank=True)
+    version_created = models.DateTimeField(null=True, blank=True)
+    url = models.TextField(max_length=2000, null=False, blank=False)
+    raw_data = JSONField(default=dict)
```

Comment:
What is the logic behind storing all of the data as both structured fields on the model and as a raw data json blob?  For fields that we rely on showing in the application it makes total sense to have them as structured columns. But for all the fields we aren't going to return or display, there is a question of whether we should be storing them at all. I can understand wanting to keep the full record, but it seems like a lot of database usage for not a lot of value, especially if its being stored twice. I think either we should
- get rid of any column that isn't going to be used by the client side and just keep all the data in raw_data
- keep data we think might be important as structured columns the way they are now but get rid of the raw_data field entirely
- keep data we think might be important as structured columns the way they are now but remove those values from the raw_data field, and only keep values in raw_data that are not otherwise stored

Reply:
Accepting option 2 - The structured columns already anticipate known future needs so removing `raw_data` entirely makes sense.